libs/grout/src/grout.test.ts

```diff
@@ -303,14 +304,10 @@ test('can send timeout', async () => {
 });
 
 test('client errors if response is not JSON', async () => {
-  const api = createApi({
-    async getStuff(): Promise<number> {
-      return 42;
-    },
-    async getMoreStuff(): Promise<number> {
-      return 42;
-    },
-  });
```

Comment:
Small preference to leave the grout tests as they were since they are using grout in the way it's intended to be used (calling `createApi` and then using `typeof` to get its type)